### DISCUSSION TITLE
How to get the drop down navigation button?

### DISCUSSION THREAD

**Discussion by compscibro** [thuliteio/doks#1113] [Q&A]
Hello, thank you so much for this amazing theme.

I am using the Doks Starter theme and trying to get a navigation drop down button like this:

<img width="833" alt="Screenshot 2023-10-11 at 12 41 12 PM" src="https://github.com/gethyas/doks/assets/69389660/ff1715a4-b036-43f5-8aaa-e7fef614f18e">

How can I achieve this?

Thanks you so much!

**Comment by h-enk** [comment]
I'm working on the new docs. For now, see the old docs —  [Add to main menu](https://v0-4-2--getdoks.netlify.app/docs/recipes/menus/#add-to-main-menu)
